src/Collectors/VersionedCollector.php

```diff
@@ -162,9 +162,11 @@ protected function getRecordsForDeletion(array $classes): array
         $keepLimit = (int) $this->config()->get('keep_limit');
         $recordLimit = (int) $this->config()->get('deletion_record_limit');
         $keepUnpublishedDrafts = (bool) $this->config()->get('keep_unpublished_drafts');
-        $deletionDate = DBDatetime::create_field('Datetime', DBDatetime::now()->Rfc2822())
```

Comment:
Would `$deletionDate = DBDatetime::now();` here do the same job?

Reply:
I think it would, I don't see why it wouldn't personally (maybe the original project I scraped this from was doing something additional). Let's see if tests fail if we make this change.

Reply:
Apparantly it does make a difference, Tests are failing after using `DBDatetime::now()` with unexpected results in the collector (e.g. More records being deleted than expected).

Which if I'm being honest, makes no sense as `now` returns the same thing (unless a mock_now) has been set. So maybe its an issue with mock times in PHPUnit tests and Silverstripe field.

Reply:
E.g.

```
$deletionDate = DBDatetime::create_field('Datetime', DBDatetime::now()->Rfc2822());
$deletionDate = $deletionDate->setValue(strtotime(
                sprintf('- %d days', $this->config()->get('keep_lifetime')),
                $deletionDate->getTimestamp()
            ))->Rfc2822();
var_dump($deletionDate);
```
Result: string(19) "2021-03-03 04:30:24"
**Tests Passing**

```
$deletionDate = DBDatetime::now()->setValue(strtotime(
                sprintf('- %d days', $this->config()->get('keep_lifetime')),
                DBDatetime::now()->getTimestamp()
            ))->Rfc2822();
var_dump($deletionDate);
```
Result: string(19) "2021-03-03 04:30:24"
**Tests Failing**

I am a bit stumped on why this change would cause tests to fail.

Reply:
Interesting and annoying at the same time. Let's keep it as it was, not a huge value in spending time trying to resolve that.